core/src/test/java/com/linecorp/armeria/client/retry/BackoffTest.java

```diff
@@ -71,4 +77,31 @@ public void withMaxAttempts() throws Exception {
         assertThat(backoff.nextIntervalMillis(2)).isEqualTo(-1);
         assertThat(backoff.nextIntervalMillis(3)).isEqualTo(-1);
     }
+
+    @Test
+    public void limitRetryAttempts() {
+        @SuppressWarnings("unchecked")
+        final Client<Request, Response> client = Mockito.mock(Client.class);
```

Comment:
Well, I actually got it from `RetryingRpcClientTest`, and it's the only place that `Mockito` is used in that way. I will fix that as well later when I pr http retrying client.